jenkins/update-opm-data.sh

```diff
@@ -55,4 +55,21 @@ $WORKSPACE/deps/opm-simulators/tests/update_reference_data.sh $OPM_DATA_ROOT
 # Finally open the pull request
 cd $OPM_DATA_ROOT
 git remote add jenkins4opm git@github.com:jenkins4opm/opm-data
+
+# Do some cleaning of old remote branches
+BRANCHES=`git branch --sort=committerdate -r | grep jenkins4opm`
+NBRANCHES=`git branch --sort=committerdate -r | grep jenkins4opm | wc -l`
+if test $NBRANCHES -gt 30
+then
+  for BRANCH in $BRANCHES
+  do
+    echo git push jenkins4pm $BRANCH:
```

Comment:
jenkins4opm perhaps?

Also, I usually do "git push origin :$BRANCH" to empty a branch, what happens with the colon after the branch name?

Reply:
doh. was done in a bit of a hurry, both things fixed.